packages/node/src/integrations/anr/common.ts

```diff
@@ -21,6 +21,16 @@ export interface Options {
    * This uses the node debugger which enables the inspector API and opens the required ports.
    */
   captureStackTrace: boolean;
+  /**
+   * Tags to include with ANR events.
+   */
+  tags: { [key: string]: Primitive };
```

Comment:
Is there a case to be made for making this some sort of hook instead of a static option? If this were a hook people could dynamically choose what tags to send. Maybe this is something we want? Maybe we even make it possible for people to access the `context`...

Reply:
The event is created in the ANR worker thread and when the event occurs, the app main thread is blocked and could be blocked indefinitely so there is no reliable way for us to get tags from customer callback.

We could hook scope updates and pass updated tags to the worker, but we would have issues with async context. 

We DO grab the trace context from the main app thread via the debugger and I suppose we could grab other context here (ie. tags, user, etc) but that is probably out of scope of this PR.

Reply:
So... maybe rename this to `staticTags`?

Reply:
`staticTags` or `anrTags` makes sense to me.